avaframe/ana5Utils/distanceTimeAnalysisCfg.ini

```diff
@@ -0,0 +1,50 @@
+### Config File - This file contains the main settings for the range time diagram in distanceTimeAnalysis
+## Copy to  local_distanceTimeAnalysisCfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# result type used for creating range time diagram e.g. FD, FV, ....
+rangeTimeResType = FD
+# time steps used for creating diagram
+tSteps = 0:1
```

Comment:
Mh, maybe it would be better to rename this to something like 'distTimeTsteps` to avoid potential confusion with the tSteps from the com1DFA ini?

Reply:
changed to distancetSteps